system_test/aest_perf_SUITE.erl

```diff
@@ -160,6 +154,16 @@ cluster(Names, Spec) -> [spec(N, Names -- [N], Spec) || N <- Names].
 spec(Name, Peers, Spec) ->
     maps:merge(maps:merge(?BASE, Spec), #{name => Name, peers => Peers}).
 
+sync_node(Node, Height, Cfg) ->
+    start_node(Node, Cfg),
+    Start = erlang:system_time(millisecond),
+    wait_for_startup([Node], 0, Cfg),
+    wait_for_sync([Node], Height, Cfg),
+    Block = get_block(Node, Height, Cfg),
+    End = erlang:system_time(millisecond),
```

Comment:
Shall end time be taken before fetching block?

Reply:
Good point.